[2024-07-13T01:35:36.501+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-13T01:35:36.519+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dnd_classification_models.hyperopt_experiment manual__2024-07-13T01:35:15.726648+00:00 [queued]>
[2024-07-13T01:35:36.524+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dnd_classification_models.hyperopt_experiment manual__2024-07-13T01:35:15.726648+00:00 [queued]>
[2024-07-13T01:35:36.525+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-13T01:35:36.543+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hyperopt_experiment> on 2024-07-13 01:35:15.726648+00:00
[2024-07-13T01:35:36.554+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=131) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-13T01:35:36.555+0000] {standard_task_runner.py:63} INFO - Started process 134 to run task
[2024-07-13T01:35:36.555+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dnd_classification_models', 'hyperopt_experiment', 'manual__2024-07-13T01:35:15.726648+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/sklearn_ml_pipeline.py', '--cfg-path', '/tmp/tmp6e7e9pzd']
[2024-07-13T01:35:36.556+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask hyperopt_experiment
[2024-07-13T01:35:36.598+0000] {task_command.py:426} INFO - Running <TaskInstance: dnd_classification_models.hyperopt_experiment manual__2024-07-13T01:35:15.726648+00:00 [running]> on host 64ae5f811ff0
[2024-07-13T01:35:36.698+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dnd_classification_models' AIRFLOW_CTX_TASK_ID='hyperopt_experiment' AIRFLOW_CTX_EXECUTION_DATE='2024-07-13T01:35:15.726648+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-13T01:35:15.726648+00:00'
[2024-07-13T01:35:36.699+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-13T01:35:36.728+0000] {logging_mixin.py:188} WARNING - 2024/07/13 01:35:36 INFO mlflow.tracking.fluent: Experiment with name 'HPO-DnD-classification' does not exist. Creating a new experiment.
[2024-07-13T01:35:36.742+0000] {logging_mixin.py:188} INFO -   0%|          | 0/10 [00:00<?, ?trial/s, best loss=?]
[2024-07-13T01:35:36.767+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.004518 seconds
[2024-07-13T01:35:36.768+0000] {tpe.py:900} INFO - TPE using 0 trials
[2024-07-13T01:35:37.346+0000] {logging_mixin.py:188} WARNING - 2024/07/13 01:35:37 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2024-07-13T01:35:38.042+0000] {logging_mixin.py:188} INFO -  10%|#         | 1/10 [00:01<00:11,  1.30s/trial, best loss: 0.36044071353620144]
[2024-07-13T01:35:38.051+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.007809 seconds
[2024-07-13T01:35:38.054+0000] {tpe.py:900} INFO - TPE using 1/1 trials with best loss 0.360441
[2024-07-13T01:35:38.270+0000] {logging_mixin.py:188} INFO -  20%|##        | 2/10 [00:01<00:05,  1.49trial/s, best loss: 0.36044071353620144]
[2024-07-13T01:35:38.280+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.009040 seconds
[2024-07-13T01:35:38.282+0000] {tpe.py:900} INFO - TPE using 2/2 trials with best loss 0.360441
[2024-07-13T01:35:38.429+0000] {logging_mixin.py:188} INFO -  30%|###       | 3/10 [00:01<00:03,  2.29trial/s, best loss: 0.36044071353620144]
[2024-07-13T01:35:38.434+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.005032 seconds
[2024-07-13T01:35:38.435+0000] {tpe.py:900} INFO - TPE using 3/3 trials with best loss 0.360441
[2024-07-13T01:35:38.660+0000] {logging_mixin.py:188} INFO -  40%|####      | 4/10 [00:01<00:02,  2.81trial/s, best loss: 0.34942287513116477]
[2024-07-13T01:35:38.664+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.003461 seconds
[2024-07-13T01:35:38.664+0000] {tpe.py:900} INFO - TPE using 4/4 trials with best loss 0.349423
[2024-07-13T01:35:38.906+0000] {logging_mixin.py:188} INFO -  50%|#####     | 5/10 [00:02<00:01,  3.17trial/s, best loss: 0.34942287513116477]
[2024-07-13T01:35:38.911+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.004430 seconds
[2024-07-13T01:35:38.911+0000] {tpe.py:900} INFO - TPE using 5/5 trials with best loss 0.349423
[2024-07-13T01:35:39.301+0000] {logging_mixin.py:188} INFO -  60%|######    | 6/10 [00:02<00:01,  2.92trial/s, best loss: 0.3389296956977964] 
[2024-07-13T01:35:39.305+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.003433 seconds
[2024-07-13T01:35:39.305+0000] {tpe.py:900} INFO - TPE using 6/6 trials with best loss 0.338930
[2024-07-13T01:35:39.772+0000] {logging_mixin.py:188} INFO -  70%|#######   | 7/10 [00:03<00:01,  2.60trial/s, best loss: 0.3389296956977964]
[2024-07-13T01:35:39.776+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.003800 seconds
[2024-07-13T01:35:39.777+0000] {tpe.py:900} INFO - TPE using 7/7 trials with best loss 0.338930
[2024-07-13T01:35:39.877+0000] {logging_mixin.py:188} INFO -  80%|########  | 8/10 [00:03<00:00,  3.38trial/s, best loss: 0.3389296956977964]
[2024-07-13T01:35:39.881+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.003394 seconds
[2024-07-13T01:35:39.882+0000] {tpe.py:900} INFO - TPE using 8/8 trials with best loss 0.338930
[2024-07-13T01:35:40.230+0000] {logging_mixin.py:188} INFO -  90%|######### | 9/10 [00:03<00:00,  3.19trial/s, best loss: 0.3389296956977964]
[2024-07-13T01:35:40.235+0000] {tpe.py:864} INFO - build_posterior_wrapper took 0.004822 seconds
[2024-07-13T01:35:40.236+0000] {tpe.py:900} INFO - TPE using 9/9 trials with best loss 0.338930
[2024-07-13T01:35:40.477+0000] {logging_mixin.py:188} INFO - 100%|##########| 10/10 [00:03<00:00,  3.41trial/s, best loss: 0.3389296956977964]
[2024-07-13T01:35:40.478+0000] {logging_mixin.py:188} INFO - 100%|##########| 10/10 [00:03<00:00,  2.68trial/s, best loss: 0.3389296956977964]
[2024-07-13T01:35:40.479+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-13T01:35:40.479+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-13T01:35:40.490+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dnd_classification_models, task_id=hyperopt_experiment, run_id=manual__2024-07-13T01:35:15.726648+00:00, execution_date=20240713T013515, start_date=20240713T013536, end_date=20240713T013540
[2024-07-13T01:35:40.610+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-13T01:35:40.635+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-13T01:35:40.647+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
